core/services/ocr2/plugins/median/plugin.go

```diff
@@ -71,6 +71,7 @@ func (m *Median) GetPluginFactory() (ocr2types.ReportingPluginFactory, error) {
 			m.runResults,
 		),
 		JuelsPerFeeCoinDataSource: ocrcommon.NewInMemoryDataSource(m.pipelineRunner, m.jb, juelsPerFeeCoinPipelineSpec, m.lggr),
+		OnchainConfigCodec:        median.StandardOnchainConfigCodec{},
```

Comment:
can we have this retrieve the OnchainConfigCodec from the relayer? just thinking ahead, but starknet has it's own custom codec

Reply:
Seems OK to wait, unless you think this would be overlooked when that times come. In which case we could stub this out a bit more at least.

Reply:
Update: this will be addressed in a future PR to not block current dev work